VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/hearing-controls/hearing-controls.component.ts

```diff
@@ -127,35 +128,46 @@ export class HearingControlsComponent implements OnInit, OnDestroy {
     }
 
     toggleMute() {
-        const muteAudio = this.videoCallService.toggleMute();
-        this.logger.info('[HearingControls] - Toggling participant mute status to' + muteAudio);
+        this.logger.info(`${this.loggerPrefix} Participant is attempting to toggle own mute status to ${!this.audioMuted}`);
+        const muteAudio = this.videoCallService.toggleMute(this.conferenceId, this.participant.id);
+        this.logger.info(`${this.loggerPrefix} Participant mute status updated to ${muteAudio}`);
         this.audioMuted = muteAudio;
     }
 
     toggleView(): boolean {
+        this.logger.info(`${this.loggerPrefix} Participant turning self-view ${this.selfViewOpen ? 'off' : 'on'}`, {
+            conference: this.conferenceId,
+            participant: this.participant.id
+        });
         return (this.selfViewOpen = !this.selfViewOpen);
     }
 
     toggleHandRaised() {
         if (this.handRaised) {
```

Comment:
do we need an attempting to toggle (pre) like the mute?

Reply:
Not necessary here since it's just a class is added/removed